benchexec/tools/vampire.py

```diff
@@ -0,0 +1,128 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for Vampire.
+    https://github.com/vprover/vampire
+    """
+
+    def name(self):
+        return "Vampire"
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("vampire")
+
+    def version(self, executable):
+        line = self._version_from_tool(executable, line_prefix="Vampire")
+        line = line.strip()
+        line = line.split(" ")[0]
+        return line.strip()
+
+    def environment(self, executable):
+        return {"keepEnv": {"TPTP": 1}}
+
+    def cmdline(self, executable, options, task, rlimits):
+        assert len(task.input_files) <= 1, "only one input file supported"
```

Comment:
Is a run with no input files actually supported?

If not, it would be easier to replace `*task.input_files` at the end of this function with `task.single_input_file` and remove this `assert`. Then there would even be a nice error message for users instead of a crash.